CONTRIBUTING.md

```diff
@@ -1,7 +1,32 @@
-Contributions to the Web Performance repositories are intended to become part of Recommendation-track documents governed by the
+## Editing Process
+
+Everyone use the standard fork, branch, and pull request workflow to propose changes to the specification. Please make branch names informative - by including the issue or bug number for example.
+
+Editorial changes that improve the readability of the specification or correct spelling or grammatical mistakes are welcome.
+
+All pull requests must have been reviewed by one or more participants of the working groups. Once the pull request gets enough support, it can be merged.
+
+### Test driven
+
+ALL normative spec changes are generally expected to have a corresponding pull request in [web-platforms-tests][WPT], either in the form of new tests or modifications to existing tests, or must include the rational for the lack of testing.
+
+Typically, both pull requests will be merged at the same time. If a pull request for the specification
```

Comment:
I think we should spell out what both means for clarity: "both pull requests (spec updates and tests)..."